client/state/action-types.js

```diff
@@ -1117,6 +1117,7 @@ export const USER_SETTINGS_UNSAVED_CLEAR = 'USER_SETTINGS_UNSAVED_CLEAR';
 export const USER_SETTINGS_UNSAVED_REMOVE = 'USER_SETTINGS_UNSAVED_REMOVE';
 export const USER_SETTINGS_UNSAVED_SET = 'USER_SETTINGS_UNSAVED_SET';
 export const USER_SETTINGS_UPDATE = 'USER_SETTINGS_UPDATE';
+export const USER_SETTINGS_UPDATE_FAILURE = 'USER_SETTINGS_UPDATE_FAILURE ';
```

Comment:
Nit: we have extra trailing space:

```suggestion
export const USER_SETTINGS_UPDATE_FAILURE = 'USER_SETTINGS_UPDATE_FAILURE';
```

Reply:
did this while cleaning up other parts of the code via a2bfdcda70fc5a29a26ca89f3ce3fcde6ccde67b